src/Home/Home.js

```diff
@@ -29,6 +30,7 @@ class Home extends Component {
                <ModalSignIn />
                <ModalNewForm />
                <ModalSignUp />
+               <ModalPhotoDisplay />
```

Comment:
How about instead of adding all these Modal components to the Home page, you just have one `ModalContainer` that abstracts away the modal stuff? Then, ModalContainer can be like, if the modalType is photos, render the Photo one